server/src/main/java/org/apache/druid/server/metrics/MetricsModule.java

```diff
@@ -210,4 +210,11 @@ public OshiSysMonitor getOshiSysMonitor(DataSourceTaskIdHolder dataSourceTaskIdH
       return new OshiSysMonitor(dimensions);
     }
   }
+
+  @Provides
+  @ManageLifecycle
+  public ServiceStatusMonitor getServiceStatusMonitor()
```

Comment:
In several services (Coordinator, MiddleManager, Peon, Overlord), `ServiceStatusMonitor` is already bound. Here, in the Metrics module, it’s explicitly bound again. We need to verify if this explicit binding overrides the existing bindings.

Reply:
The bindings seem to be singleton so I don't think it's overriding or double-injecting. For example the logs aren't showing 2 heartbeats being emitted. Let me know if there's another place I could be checking.